packages/vscode-extension/src/error.ts

```diff
@@ -16,4 +16,6 @@ export enum ExtensionErrors {
   FolderAlreadyExist = "FolderAlreadyExist",
   InvalidProject = "InvalidProject",
   FetchSampleError = "FetchSampleError",
+  OpenEnvProfileError = "OpenEnvProfileError",
+  NoEnvProfileError = "NoEnvProfileError",
```

Comment:
how about replacing it with `EnvProfileNotFoundError`?

Reply:
Fixed.